src/bp/nlu-core/training-pipeline.ts

```diff
@@ -531,29 +507,18 @@ const TrainSlotTagger = async (input: TrainStep, tools: Tools, progress: progres
   debugTraining.forBot(input.botId, 'Training slot tagger')
   const slotTagger = new SlotTagger(tools.mlToolkit)
 
-  try {
-    await slotTagger.train(
-      input.intents.filter(i => i.name !== NONE_INTENT),
-      () => progress(0) // not increasing actual progress but checking if training was cancelled
-    )
-  } catch (err) {
-    if (err instanceof TrainingCanceledError) {
-      return
-    }
-    throw err
-  }
+  await slotTagger.train(
+    input.intents.filter(i => i.name !== NONE_INTENT),
+    () => progress(0) // not increasing actual progress but checking if training was cancelled
```

Comment:
nice hack, but is this even necessary since the training process gets killed

Reply:
nope, you're right! good catch